packages/cli/src/commands/push.ts

```diff
@@ -34,12 +34,13 @@ export type PushOptions = {
   destination?: string;
   branchName?: string;
   upsert?: boolean;
-  'batch-id'?: string;
+  'job-id'?: string;
   'batch-size'?: number;
   region?: Region;
   'skip-decorator'?: string[];
```

Comment:
I know we didn't introduce this in this change - but why is the skip-decorator option here? I assume the push command doesn't do anything other than push ... or does it also lint/bundle/decorate/something else ?

Reply:
Because we bundle a definition before push it

Reply:
Ah, right. And bundle still also decorates as a side effect, is that right? Do we need `skip-rule`, `skip-preprocessor` and the other options also?

Reply:
We definitely do not need `skip-rules`, but I am not sure regarding `skip-preproccesors` because  we recommend avoid preprocessing https://redocly.com/docs/cli/resources/custom-plugins/#preprocessors